.storybook/addons.js

```diff
@@ -0,0 +1 @@
+import '@kadira/storybook-addon-links/register'
```

Comment:
This file overrides the default storybook addons to remove the unused bottom  panel and only include the sidebar links.